fem/bilininteg.hpp

```diff
@@ -1678,6 +1678,8 @@ class DiffusionIntegrator: public BilinearFormIntegrator
    const GeometricFactors *geom;  ///< Not owned
    int dim, ne, dofs1D, quad1D;
    Vector pa_data;
+   // CEED extension
+   void* ceedDataPtr;
```

Comment:
Will this make libceed a required mfem dependency?

Reply:
Absolutely nothing is stored or precomputed, everything is computed on the fly.

Reply:
No, you have to compile with `MFEM_USE_CEED=YES` to use libCEED. I could guard this `void*` with `#ifdef MFEM_USE_CEED`, but a `void*` is not a dependency issue by itself.